docs/topics/serialization.txt

```diff
@@ -268,6 +268,15 @@ work::
                 return force_text(obj)
             return super(LazyEncoder, self).default(obj)
 
+By default, :class:`~django.core.serializers.json.DjangoJSONEncoder` is used
+for JSON encoding/decoding.  It should also be used as a parent class when
+creating your own encoding class, as in the example above.
+:class:`~django.core.serializers.json.DjangoJSONEncoder` will handle
+:class:`~datetime.datetime`, :class:`~datetime.date`, :class:`~datetime.time`,
+:class:`~datetime.timedelta`, :class:`~decimal.Decimal`, :class:`~uuid.UUID`,
```

Comment:
OK.  I wasn't sure about that because it seemed like that pull request could potentially be merged before this one made it.

Also, should Promise be included in the list?  It doesn't seem to be documented anywhere, so I can't create a link to anywhere for it.